launcher/net/NetAction.h

```diff
@@ -35,18 +36,38 @@
 
 #pragma once
 
+#include <cmath>
+
 #include <QNetworkReply>
 #include <QUrl>
 
 #include "QObjectPtr.h"
 #include "tasks/Task.h"
 
+static const QStringList s_units_si  {"kb", "MB", "GB", "TB"};
+static const QStringList s_units_kibi {"kiB", "MiB", "Gib", "TiB"};
```

Comment:
```suggestion
static const QStringList s_units_si  {"kB", "MB", "GB", "TB"};
static const QStringList s_units_kibi {"KiB", "MiB", "Gib", "TiB"};
```

<details>
<summary>kibi</summary>

![kibe](https://user-images.githubusercontent.com/9145768/236562733-5cf346d3-adce-4144-bcdf-bc9b8ecd6a54.png)

</details>

Reply:
I know there is a more technical name for the unit system but...  ~kibi bytes~